### PR TITLE
Multi-Plugin Example

### PR DESCRIPTION
**Pull request by nichelle-hall**
An example of using multiple plugins to gather specific measurements and publish them to Grafana through some persistence layer (Graphite or Influxdb).

Summary of changes:
- Added another folder in the examples directory complete with Dockerfiles, tasks, scripts, and Grafana dashboard that will be utilized when the users runs the example

### CODE REVIEW

**Comment by mjbrender** [comment]
I really like this idea of an example @NichelleBot. Comments will be in-line. We may adjust this as we scope the examples folder further based on #1158, but that shouldn't stop this from merging.

File: examples/multi-plugins/README.md [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
+  - Gets, loads and starts snap plugins
+  - Creates and starts a task that collects from psutil, cpu, and cgroups then publishes to InfluxDB
+  
+Start the example by running `run-multi.sh`
+
+### Requirements
+- docker
+- docker-compose 
+- netcat 
+- SNAP_PATH env var set to the snap build directory
```
Comment (mjbrender) [inline examples/multi-plugins/README.md]: this demo you're using as a baseline is working on a revision. That shouldn't be an issue.

**Comment by mjbrender** [comment]
@NichelleBot can you squash down these commits? Thanks!

**Comment by jcooklin** [comment]
@NichelleBot and I have talked about this PR **not** requiring netcat, SNAP_PATH, or building anything to work.  

We also discussed how this example can be integrated into a large e2e test as part of another issue.

**Comment by candysmurf** [comment]
@jcooklin, should this example be in this repo?

**Comment by jcooklin** [comment]
@candysmurf, not as it is currently formed but in general yes.  We discussed making the following changes.

- [ ] Remove the need for compiling anything to run the example
 - A requirement to run the example(s) will be that docker-compose is available and configured
- [ ] Add a large (end-2-end) test that tests the example 
 - Configure CI to run the e2e test

**Pull request closed without merging**